### ISSUE TITLE
[Question] Is there a way to export lists as CSV?

### ISSUE THREAD

**Issue by Guts** [halcy/Mastodon.py#369]
Hi there,

I was wondering if there is a way to export lists into CSV using this API wrapper in the same way it's possible through the web client?

**Comment by halcy** [comment]
Not entirely sure what you mean. Lists of followers / following / blocked etc.? There is no native, one-shot way to get a CSV download, but probably you could build something that exports the same data

**Comment by Guts** [comment]
Yes, I'm reffering to the Import/export menu in Mastodon web application settings: https://docs.joinmastodon.org/user/moving/#export

![image](https://github.com/halcy/Mastodon.py/assets/1596222/802d1e76-0583-4625-86a7-3dc96440da62)

It allows to export following accounts, lists of accounts. Then it's can shared for example on something like https://trutzig89182.github.io/Mastodon-Sociologists/ (or just sending csv obvioulsy).

As you say, it's fairly simple to export lists of accounts using mastodon.py ([my script in dev](https://github.com/geotribu/cli/blob/feature/export-mastodon/tests/dev/dev_mastodon_py.py) if someone is looking for the same thing).

But, there are some information is present in Mastodon UI's export but missing in account object through Mastodon.py:

- `Show boost`
- `Languages`
- `Notify on new posts`

**Comment by halcy** [comment]
I think unfortunately that isn't possible, so until Mastodon adds that as a function, this is a wont (can't) fix.
